etn-ccis/blui-react-workflows
# Feature/4172 implement login screen base

Fixes BLUI-4172.



#### Changes proposed in this Pull Request:

- Add login screen base
- Add login screen base tests
- Add login screen example to the example 2 test app



#### Screenshots / Screen Recording (if applicable)
![Screen Shot 2023-05-24 at 5 59 17 PM](https://github.com/etn-ccis/blui-react-workflows/assets/13989985/e824da34-74d4-466f-ac33-f67d0f071513)



#### To Test:

- `yarn start:example2`
- ensure everything works as expected according to the acceptance criteria.



#### Any specific feedback you are looking for?

- I added some error handling inside of this component that may be handled differently or set up and managed at a higher level in the component hierarchy as we build this out. That being said, do not worry about the error message box, although it is configurable and testable if you want to check it out. Also, ignore anything in regards to the ErrorDialog. I want to leave the code there as we will use something similar at some point, but for now, I want to have more discussion around error handling and get this in as is so we can start hooking up routing and the proper setup for the demo project.
